src/Our.Umbraco.StackedContent/Web/UI/App_Plugins/StackedContent/views/stackedcontent.html

```diff
@@ -40,7 +42,8 @@ <h3>{{itm.name}}</h3>
             </div>
 
             <div class="stack__add-bar stack__add-bar--bottom" ng-if="model.config.singleItemMode !== '1'">
-                <a ng-click="addContent($event, model.value.length)" class="stack__add-button" ng-if="canAdd()"><i class="icon icon-add"></i></a>
+                <a ng-click="addContent($event, model.value.length)" class="stack__add-button" ng-if="canAdd()" title="Add"><i class="icon icon-add"></i></a>
+                <a ng-click="pasteFromLocalStorage($event, model.value.length)" class="stack__paste-button" ng-if="canPaste()" title="Paste"><i class="icon icon-documents"></i></a>
```

Comment:
Not sure if we should go for more of a clipboard icon for paste? That seems to be the tradition.

Reply:
Hey @mattbrailsford I was hoping to use one of the default icons that Umbraco provides. I've looked the list over a time or two and don't see anything for a clipboard. Is there a better alternative - or would you rather a custom icon be added to the project?

Reply:
I think there should be an `icon-paste-in` that should do the job?

Reply:
Got it!